src/hooks/useScreenSize.ts

```diff
@@ -0,0 +1,75 @@
+import { useState, useEffect } from 'react';
+
+type ScreenSizes = {
+  // 2xs
+  txs: boolean;
+  xs: boolean;
+  sm: boolean;
+  md: boolean;
+  lg: boolean;
+  xl: boolean;
+  // 2xl
+  txl: boolean;
+};
+
+const breakpoints: { [key in keyof ScreenSizes]: string } = {
+  txs: '(min-width: 320px)',
+  xs: '(min-width: 375px)',
+  sm: '(min-width: 640px)',
+  md: '(min-width: 768px)',
+  lg: '(min-width: 1024px)',
+  xl: '(min-width: 1280px)',
+  txl: '(min-width: 1536px)',
+};
```

Comment:
훅 너무 좋네요!! 그런데 혹시 breakpoint를 관리하는 상수 하나 두시는건 어떻게 생각하시나요?!
```ts
// src/constants
const BREAK_POINT = {
  txs : 320,
  xs : 375,
  //...
}
```
`useScreenSize`훅에서는 해당 상수를 가져다가 쓰는 방식으로 하는건 별로일까요?

```suggestion
type BreakpointValues = {
    [key in keyof typeof BREAK_POINT]: string;
};

const breakpoints: BreakpointValues = Object.keys(BREAK_POINT).reduce((acc, key) => {
    acc[key] = `(min-width: ${BREAK_POINT[key]}px)`;
    return acc;
}, {} as BreakpointValues);
};
```

Reply:
생각해보니 breakpoint 상수를 만들어도 쓸일이 뭔가 딱히 없을거 같기도 하네요(?) 꼭 저렇게 안해도 괜찮을거 같습니당!

Reply:
요것도 방법인 것 같아여! 테일윈드 설정 파일에서도 저 상수 참조하면 좋을 것 같은데, 일단은 방법을 모르겠어서 이번에는 욜케해두겠음당! 코멘트 감사함당!